include/deal.II/non_matching/coupling.h

```diff
@@ -221,6 +222,135 @@ namespace NonMatching
     const ComponentMask &          immersed_comps = ComponentMask(),
     const Mapping<dim1, spacedim> &immersed_mapping =
       StaticMappingQ1<dim1, spacedim>::mapping);
+
+  /**
+   * Create a coupling sparsity pattern for non-matching independent grids,
+   * using a convolution kernel with compact support of radius epsilon.
+   *
+   * Given two non-matching triangulations, representing the domains $\Omega^0$
+   * and $\Omega^1$, both embedded in $R^d$, and two finite element spaces
+   * $V^0(\Omega^0) = \text{span}\{v_i\}_{i=0}^n$ and $V^1(\Omega^1) =
+   * \text{span}\{w_\alpha\}_{\alpha=0}^m$, compute the sparsity pattern that
```

Comment:
We're not using j on porpuse, since people associate to `ij` a square matrix. Here `i` runs up to `n`, while `alpha` runs up to `m`. Explicitly distinguishing the index types, make people alert about the difference between B, and B^T.